manifests/augeas.pp

```diff
@@ -38,12 +38,12 @@
 define jenkins::augeas (
   String $config_filename,
   Variant[Array[String], String] $changes,
-  Optional[Variant[Array[String], String]] $onlyif = undef,
-  Optional[String] $plugin_version                 = undef,
-  String $context                                  = '/',
-  Optional[Variant[Boolean,String]] $plugin        = false,
-  Boolean $restart                                 = false,
-  Boolean $show_diff                               = true,
+  Optional[Variant[Array[String], String]] $onlyif         = undef,
```

Comment:
I'll say that personally I think aligning this way is a bad pattern. It only causes huge diffs IMHO.

Reply:
reverted